### PR TITLE
[17.0][MIG] partner_invoicing_mode: Migration to 17.0

### PR DESCRIPTION
**Pull request by NICO-SOLUTIONS**
standard migration

### CODE REVIEW

File: partner_invoicing_mode/readme/DESCRIPTION.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+This is a base module for implementing different invoicing mode for
+customers. It adds a selection field invoicing_mode in the Accounting
+tab of the partner with a default value (Odoo standard invoicing mode).
```
Comment (traitrantrobz) [inline partner_invoicing_mode/readme/DESCRIPTION.md]: I think this needs to be fixed. In Odoo version 17.0, the tab name 'Accounting' has been changed to 'Invoicing'.

**Review by traitrantrobz — CHANGES_REQUESTED** [review]

File: partner_invoicing_mode/readme/DESCRIPTION.md (hunk 1 above)
Comment (NICO-SOLUTIONS) [reply partner_invoicing_mode/readme/DESCRIPTION.md]: adjusted

**Review by traitrantrobz — CHANGES_REQUESTED** [review]
@NICO-SOLUTIONS I think you should double-check against this migration. You can check at #1515 and #1295.

**Comment by traitrantrobz** [comment]
@NICO-SOLUTIONS You missed a lot of commit history. Please rebase the commit and migration against it.
Ảnh chụp Màn hình 2024-07-30 lúc 15 48 10

File: partner_invoicing_mode/views/res_config_settings.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                <setting
+                    id="invoicing_mode"
+                    string="Invoicing Mode"
+                    help="Standard Invoicing Options"
+                >
+                    <div class="mt16">
+                        <div class="content-group" id="standard_invoicing_settings">
+                            <div>
+                                <field name="invoicing_mode_standard_last_execution" />
+                                <label
```
Comment (traitrantrobz) [inline partner_invoicing_mode/views/res_config_settings.xml:18]: Please double-check position of `invoicing_mode_standard_last_execution `.
Ảnh chụp Màn hình 2024-08-06 lúc 10 48 50

**Review by traitrantrobz — CHANGES_REQUESTED** [review]

**Review by traitrantrobz — APPROVED** [review]
LGTM 👍

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**